tcp_check/tests/common.py

```diff
@@ -14,3 +14,5 @@ def _test_check(aggregator):
     expected_tags = ['foo:bar', 'target_host:datadoghq.com', 'port:80', 'instance:UpService']
     aggregator.assert_metric('network.tcp.can_connect', value=1, tags=expected_tags)
     aggregator.assert_service_check('tcp.can_connect', status=TCPCheck.OK, tags=expected_tags)
+    aggregator.assert_all_metrics_covered()
+    assert len(aggregator._service_checks['tcp.can_connect']) == 1
```

Comment:
Why not add count to:

```python
    aggregator.assert_service_check('tcp.can_connect', status=TCPCheck.OK, tags=expected_tags, count=1)
```

Instead of 

```python
assert len(aggregator._service_checks['tcp.can_connect']) == 1
```

Same comment for the other cases in this PR.

Reply:
Because the expected duplicates have different statuses (OK/CRITICAL).
`aggregator.assert_service_check('tcp.can_connect', status=TCPCheck.OK, tags=expected_tags, count=1)` would not catch if an extra CRITICAL service check has been submitted.

I updated the PR with Thomas suggestion